calico_containers/calico_ctl/endpoint.py

```diff
@@ -67,8 +67,8 @@ def validate_arguments(arguments):
 
     # Validate Profiles
     profile_ok = True
-    if "<PROFILES>" in arguments:
-        profiles = arguments.get("<PROFILES>")
+    profiles = arguments.get("<PROFILES>")
+    if profiles is not None:
```

Comment:
This could be `if profiles`, but the way that it is now works fine too since the value of `<PROFILES>` will be set to `None` if it is not specified, which will be interpreted as `False` by python.